### ISSUE TITLE
 humble-desktop-pytorch-l4t-r35.4.1 - Missing ros_deep_learning

### ISSUE THREAD

**Issue by omerts** [dusty-nv/jetson-containers#363]
Hello,

The docker image: `dustynv/ros:humble-desktop-pytorch-l4t-r35.4.1` is missing the ros_deep_learning package:
![image](https://github.com/dusty-nv/jetson-containers/assets/4996164/ffc398a8-ce0e-4fdb-aa71-fab3b5180d06)

**Comment by Hommus** [comment]
I came across the same issue when building as follows:
```bash
./build.sh pytorch ros:iron-perception
```

**Comment by Hommus** [comment]
@omerts More info here on how to include ros_deep_learning: https://github.com/dusty-nv/jetson-containers/issues/286

**Comment by Hommus** [comment]
@dusty-nv Is there a timeline for ros_deep_learning to be included in L4T-r35.4.1 by default? I've been trying to build a ros:iron-perception image with pytorch and ros_deep_learning but I haven't been successful yet.

**Comment by omerts** [comment]
> I came across the same issue when building as follows:
> 
> ```shell
> ./build.sh pytorch ros:iron-perception
> ```
> 
> @omerts You should be able to build the image with a previous version of L4T to fix it. Try `l4t-r35.3.1`

Yes it works with 35.3.1, but this is unrelated to the issue which is for 35.4.1.
